bitnami/minio/Chart.yaml

```diff
@@ -25,4 +25,4 @@ name: minio
 sources:
   - https://github.com/bitnami/containers/tree/main/bitnami/minio
   - https://min.io
-version: 11.9.2
+version: 11.9.3
```

Comment:
Since this is adding a new feature, could you increase the minor version? I.e. to `11.10.0`.

Reply:
ok